kinds/rolemapping/role_mapping_kind.cue

```diff
@@ -0,0 +1,23 @@
+package kind
+
+name:        "RoleMapping"
```

Comment:
Do we call this RoleBinding? If so do we tweak it to more closely resemble a standard k8s RoleBinding? We are already using the k8s Role name above (though our role definition doesn't really match up with a k8s Role type)

https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding